tools/sim/bridge.py

```diff
@@ -208,7 +208,7 @@ def destroy():
 
   vc = carla.VehicleControl(throttle=0, steer=0, brake=0, reverse=False)
 
-  is_openpilot_engaged = False
+  is_openpilot_engaged = True
```

Comment:
think openpilot should always be engaged in the sim, there's a bunch of hacks right now to make manual cotnrol work

Reply:
CARLA already has support for manual control anyway, don't think there's value in duplicating that